lms/templates/problem.html

```diff
@@ -19,7 +19,7 @@ <h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="
       <div class="problem-hint">
         <%include file="problem_notifications.html" args="
          notification_name='hint',
-         notification_type='hint',
+         notification_type='problem-hint',
```

Comment:
This is they type, similar to warning or error. I could actually just leave it blank for hint since I am now using notification-hint for the styling.